scripts/go-mod-tidy-all.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
ah, I see you don't have a makefile rule for this tho yea? Seems like it would be nice to have

Reply:
Sure, any preference qua name?

Reply:
none! `tidy-all` is what I would go for if pressed.